docs/src/spaces.md

```diff
@@ -0,0 +1,155 @@
+# [**9.** Scratchspaces](@id Scratchspaces)
+
+!!! compat "Julia 1.6"
+    Pkg's Scratchspaces functionality requires at least Julia 1.6.
+
+`Pkg` can manage and lifecycle scratchspaces of temporary or readily-recreatable data.
+These spaces can contain datasets, text, binaries, or any other kind of data that would be convenient to store, but which is non-fatal to have garbage collected if it has not been accessed recently, or if the owning package has been uninstalled.
+As compared to [Artifacts](@ref), these containers of data are mutable and should be treated as ephemeral; all usage of scratchspaces should assume that the data stored within them could be gone by the next time your code is run.
+In the current implementation, scratchspaces are removed during Pkg garbage collection if the space has not been accessed for a period of time (see the `spaces_cleanup_period` keyword argument to [Pkg.gc](@ref)), or if the owning package has been removed.
+Users can also request a full wipe of all spaces to clean up unused disk space through `Pkg.Spaces.clear_spaces!()`.
+
+## API overview
```

Comment:
I wonder if this should also mention the case when a scrach file is " shared among multiple packages."

Reply:
Or is that done by just leaving out the `pkg_uuid` arg in `get_space!`?

Reply:
Yes, precisely.  That's what is meant by a "global" space.